src/app/(default)/components/ui/Summary.tsx

```diff
@@ -0,0 +1,15 @@
+import { ReactNode } from 'react'
+
+interface Props {
+  columns: {
+    left: ReactNode
+    right?: ReactNode
+  }
+}
+
+export const Summary: React.FC<Props> = ({ columns: { left, right } }) => (
+  <div className='area-summary'>
+    <div className='area-summary-left'>{left}</div>
+    {right !== undefined ? <div className='area-summary-right'>{right}</div> : null}
```

Comment:
can you test for undefined with `!=` to catch both undefined and null?  Ex: 
```javascript
right != undefined ?
```

Reply:
yep, done!